modules/ROOT/pages/setting-up-ldap-provider-for-spring-security.adoc

```diff
@@ -0,0 +1,118 @@
+= Configure LDAP Provider for Spring Security
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, connector, spring security, spring, ldap, authentication
+
+In Mule 4, you can use a Spring Security LDAP provider to perform component authorization, or use it as a Mule security provider via the Spring Module. Before you configure the LDAP provider obtain the Spring Jar files and declare Spring Beans.
```

Comment:
```suggestion
In Mule 4, you can either use a Spring Security LDAP provider to perform component authorization, or use it as a Mule security provider via the Spring module. Before you configure the LDAP provider, obtain the Spring JAR files and declare Spring beans.
```
I currently have Spring module and Spring beans in the term presentation list. Please let me know if you have other input for Spring terminology presentation rules. I looked at this, but became overwhelmed: https://docs.spring.io/spring-security/site/docs/current/reference/htmlsingle/.